README.md

```diff
@@ -1,37 +1,110 @@
-[![Check unchanged](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/check_unchanged.yml/badge.svg)](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/check_unchanged.yml)
 [![Unit tests](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/push.yml/badge.svg)](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/push.yml)
 [![Validate Parla-Clarin XML](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/validate.yml/badge.svg)](https://github.com/welfare-state-analytics/riksdagen-corpus/actions/workflows/validate.yml)
 
-# Swedish parliamentary proceedings - Riksdagens protokoll 1867-today
 
-_Westac Project, 2020-2023_
+# Swedish parliamentary proceedings --- 1867--today --- v0.10.0
+
+_Westac Project_, 2020--2023 | 
```

Comment:
I think westac should be all the way to at least 2024.

Reply:
Yes, it will formally run throughout 2024.

Reply:
Resolved: 7bd25daa97